crates/router/src/connector/paypal/transformers.rs

```diff
@@ -1196,13 +1219,222 @@ impl From<PaypalWebhookEventType> for api::IncomingWebhookEvent {
             PaypalWebhookEventType::PaymentCaptureCompleted
             | PaypalWebhookEventType::CheckoutOrderCompleted => Self::PaymentIntentSuccess,
             PaypalWebhookEventType::PaymentCapturePending
-            | PaypalWebhookEventType::CheckoutOrderApproved
             | PaypalWebhookEventType::CheckoutOrderProcessed => Self::PaymentIntentProcessing,
             PaypalWebhookEventType::PaymentCaptureDeclined => Self::PaymentIntentFailure,
             PaypalWebhookEventType::PaymentCaptureRefunded => Self::RefundSuccess,
-            PaypalWebhookEventType::Unknown
-            | PaypalWebhookEventType::PaymentAuthorizationCreated
-            | PaypalWebhookEventType::PaymentAuthorizationVoided => Self::EventNotSupported,
+            PaypalWebhookEventType::PaymentAuthorizationCreated
+            | PaypalWebhookEventType::PaymentAuthorizationVoided
+            | PaypalWebhookEventType::CheckoutOrderApproved
+            | PaypalWebhookEventType::Unknown => Self::EventNotSupported,
         }
     }
 }
+
+#[derive(Deserialize, Serialize, Debug)]
+pub struct PaypalSourceVerificationRequest {
+    pub transmission_id: String,
+    pub transmission_time: String,
```

Comment:
Can this be a `PrimitiveDateTime`, or does it need to be a `String`?

Reply:
it needs to be as string , in this case